### ISSUE TITLE
"Unexpected token in" from Flow

### ISSUE THREAD

**Issue by gausie** [rt2zz/redux-persist#537]
I get the following Flow errors on a project for which I have redux-persist installed.

```
Error: node_modules/redux-persist/es/index.js.flow:53
 53: declare function createTransform(in: Transformer, out: Transformer, config?: TransformConfig): Transform
                                      ^^ Unexpected token in

Error: node_modules/redux-persist/lib/index.js.flow:53
 53: declare function createTransform(in: Transformer, out: Transformer, config?: TransformConfig): Transform
                                      ^^ Unexpected token in
```

I assume this is because `in` is a reserved word

**Comment by rt2zz** [comment]
this has been fixed in 5.3.4 👍

**Comment by KDederichs** [comment]
Any chance we can get a fix for that in v4?
